zenstruck/foundry/2.6/config/packages/zenstruck_foundry.yaml

```
@@ -0,0 +1,8 @@
+when@dev: &dev
+    # See full configuration: https://symfony.com/bundles/ZenstruckFoundryBundle/current/index.html#full-default-bundle-configuration
+    zenstruck_foundry:
+        persistence:
+            # Flush only once per call of `PersistentObjectFactory::create()` in userland
```

```suggestion
            # Flush only once per call of `PersistentObjectFactory::create()`
```

recipes are for userland :)

---

right. I've also fixed recipe for 2.5